### PR TITLE
Update usr_51.txt

### PR DESCRIPTION
**Pull request by cuppajoeman**
use their instead of his as the user may not be male

### CODE REVIEW

File: runtime/doc/usr_51.txt [hunk 1]
```diff
@@ -103,7 +103,7 @@ NOT LOADING
 
 It is possible that a user doesn't always want to load this plugin.  Or the
 system administrator has dropped it in the system-wide plugin directory, but a
-user has his own plugin he wants to use.  Then the user must have a chance to
+user has their own plugin he wants to use.  Then the user must have a chance to
```
Comment (k-takata) [inline runtime/doc/usr_51.txt]: "he" should be also changed?

**Comment by chrisbra** [comment]
I am just mentioning it here for reference here: https://vimhelp.org/helphelp.txt.html#inclusion

**Comment by Shane-XB-Qian** [comment]
> > I am just mentioning it here for reference here: https://vimhelp.org/helphelp.txt.html#inclusion
> 
> `:help inclusion` says
> 
> > Some people have suggested using "they",
> > but that is not regular English
> 
> Actually, it is "regular English".

@errael please donot take this too seriously,
this is a tech community it does waste many times to argu such.
// i supposed you were not english-native person too, let it be~ :smile:

-- 
shane.xb.qian

**Comment by chrisbra** [comment]
It might be regular English, but as a non-English native speaker, it sounded unusual to me (although I got used to it) and I don't remember being taught that this is regualar English.

**Comment by cuppajoeman** [comment]
I had not read the inclusion section before I made this. 

I'll leave it up to the main members to decide whether or not this should go through

**Comment by Shane-XB-Qian** [comment]
> I had not read the inclusion section before I made this. 

you can search it, it does happen some times.

> I'll leave it up to the main members to decide whether or not this should go through

not sure why this was a matter to you, but again this is a tech community,
wish or let's not make the 'main members' difficult to handle such "sensitive" but not much 'tech' kind stuff things.
// so let me try to answer you instead. :smile:

-- 
shane.xb.qian

**Comment by benknoble** [comment]
Sticking our heads in the sand and refusing to deal with it because it isn't "technical" only means we are content with the status quo and won't change it. I'm not content with the status quo of "he" as "gender neutral" in technical writing. Singular "they" is accepted and encouraged in academic and technical writing:
- https://gist.github.com/0xabad1dea/8870b192fd1758743f66
- https://apastyle.apa.org/blog/singular-they
- https://writing.stackexchange.com/q/2979/33198

The use of "they" is very established and poses practically no problems (when ambiguity is introduced, we should be writing without pronouns anyway to remove it).

I am not saying we need to overhaul all of runtime/doc right now; perhaps it could be done gradually by requiring that changes to documentation also update the pronouns of touched lines? If someone wants to take the time to update the pronouns, why stop/discourage them?

**Comment by Shane-XB-Qian** [comment]
"poses": that's an interesting word in the world. aha!

to be clear: here no one discourage something, but just same saying his/her/its /ones or my disagree / agree.

--
shane.xb.qian

**Comment by dkearns** [comment]
> It might be regular English, but as a non-English native speaker, it sounded unusual to me (although I got used to it) and I don't remember being taught that this is regualar English.

Judging by your profile picture, I have a few years on you but I was taught this usage in the mid '80s.  Use of he, he/she, or even she was openly derided by most of my teachers.

Whether we speak anything approaching regular English in Australia is open to debate.

Here's the old thread: https://github.com/vim/vim/pull/6535#discussion_r460512349

**Comment by errael** [comment]
> Here's the old thread: #6535 (comment)

Thanks. I have never seen a thread with more passionate participation; and Bram finally locking it with:
> Looks like we heard all opinions at least twice now.

One thing that's fascinating to me: a lot of the dissension, to using "they/their", comes from non-native English speakers. That may have more to say about those responsible for their curriculum, rather than native English itself. That would fit with the statement
> some grammarians, especially toward the end of the 1800s, declared that
> singular “they” is incorrect grammar.

Taken from the [entertaining article](https://www.quora.com/What-is-the-reason-why-some-people-get-offended-when-someone-uses-they-as-a-singular-pronoun-Is-it-grammatically-correct) with it's many examples from literature.

**Comment by errael** [comment]
> I had not read the inclusion section before I made this.
> 
> I'll leave it up to the main members to decide whether or not this should go through

The change suggested in https://github.com/vim/vim/pull/13496#discussion_r1384376814 should be made before this would be included. (Not to say that it would be even if the suggested change is made)

**Comment by benknoble** [comment]
It seems some mailing list comments are not being mirrored here. This is the mailing list thread for the PR if you want to subscribe: https://groups.google.com/g/vim_dev/c/ZrHIa9QMEUI

And thank you Ernie for responding to a message I had missed on the mailing list.

**Comment by chrisbra** [comment]
I know it is controversial, but if this serves the goal of being more inclusive and people feel no longer hurt, it is fine with me. However changing the whole documentation will be a huge effort. But if people are willing to do the work, than I am willing to merge (but it must serve the goal of keeping the documentation understandable).

@cuppajoeman can you please also fix the mentioned "he" from [Kens comment](https://github.com/vim/vim/pull/13496#pullrequestreview-1716815437)?

**Comment by errael** [comment]
> However changing the whole documentation will be a huge effort. But if people are willing to do the work, than I am willing to merge (but it must serve the goal of keeping the documentation understandable).

The original PR, #6535, references the [Google developer documentation style guide](https://developers.google.com/style/pronouns) on how to write gender-neutral technical documentation. 

I looked a little at that PR and compared it with what the doc is now. Instead of just substituting pronouns, the text can often be re-written such that pronouns are not required. That's also one of the suggestions I saw in at least one of the many references.

**Comment by cuppajoeman** [comment]
> I know it is controversial, but if this serves the goal of being more inclusive and people feel no longer hurt, it is fine with me. However changing the whole documentation will be a huge effort. But if people are willing to do the work, than I am willing to merge (but it must serve the goal of keeping the documentation understandable).
> 
> @cuppajoeman can you please also fix the mentioned "he" from [Kens comment](https://github.com/vim/vim/pull/13496#pullrequestreview-1716815437)?

Yes, just fixed other instances.

File: runtime/doc/usr_51.txt (hunk 1 above)
Comment (cuppajoeman) [reply runtime/doc/usr_51.txt]: Newest commit fixes this

**Comment by errael** [comment]
> […]
> 
> > One thing that's fascinating to me: a lot of the dissension, to using "they/their", comes from non-native English speakers. […]
> 
> Maybe so; but let us not forget that Vim, its developers and users, are a motley lot: I wouldn't be surprised if more than a majority (of active developers, anyway) had other mother languages than English.

Yes, during this discussion I guessed over half. That contributed to the conjecture that the educational system gave a restricted sense of "proper English".

> Bram himself was a Dutchman, Christian is German, I am Belgian, and several of the most active developers are Israeli, Russian, Chinese, Japanese, you name it; and among all these "second-language English speakers" only a few use what I would call "obviously awkward language" when writing in English.
> 
> I have come to tolerate this "plural they" use which was never used when I learnt English some 60 years or so ago, and later only in colloquial language or in militantly battle-of-all-sexes language;

Unless the references are incorrect, [Singular they](https://en.wikipedia.org/wiki/Singular_they), has been around since the 14th century. It was criticized by the experts in the 18th century and experienced a resurgence with the early 21st century move toward gender-neutral language. It is now formally accepted.

> but I am reluctant to use it myself when talking about what is obviously a single person — I would prefer either some forms like "you", "(s)he", or to turn the phrase another way so that none of _he_, _she_ or _they_ (or their derivatives) would have to be used.

I'm not a linguist or cultural anthropologist but one thing I'm curious about is how someone's native language affects their perception and use of a 2nd language. I wonder if the French, where everything has a gender, or the Germans, which don't have (AFAIK) gender neutral pronouns, are especially dismayed by rising popularity of gender-neutral language.

**Comment by Shane-XB-Qian** [comment]
> I know it is controversial, but if this serves the goal of being more inclusive and people feel no longer hurt.

I felt you already hurt yourself. ahahaha~
vim is a tech community, changing such things not sure if would improve the difference, or would say this perhaps just was not the right place (definitely there were some difference).

> Bram himself was a Dutchman, Christian is German, I am Belgian, and several of the most active developers are Israeli, Russian, Chinese, Japanese.

I am Chinese, not sure yepaggan where from.
I think the doc the main purpose should focus on clearing the meaning and giving direct specification, others not a matter unless he/she/it/one was doing it on purpose.
// yea, for example like this: he/she/it/one.
// how about changing to 'it' or 'one'. :smile:
// even it just was a 'he', supposed everyone knew it did not mean a "man" was doing that on purpose.

--
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
that called culture, you donot have to make others to accept your 'positive'.
about that word 'they' someone had answered you (in the mail) that's a not interested and perhaps wrong political correction.
// to be clear again: to me whatever you use, even let's speak Chinese (let's forget that horrible English) maybe better, :-)

--
shane.xb.qian

File: runtime/doc/usr_51.txt [hunk 2]
```diff
@@ -103,7 +103,7 @@ NOT LOADING
 
 It is possible that a user doesn't always want to load this plugin.  Or the
 system administrator has dropped it in the system-wide plugin directory, but a
-user has his own plugin he wants to use.  Then the user must have a chance to
+user has their own plugin they want to use.  Then the user must have a chance to
 disable loading this specific plugin.  These lines will make it possible: >
```
Comment (k-takata) [inline runtime/doc/usr_51.txt]:
```suggestion
user has their own plugin they want to use.  Then the user must have a chance
to disable loading this specific plugin.  These lines will make it possible: >
```

File: runtime/doc/usr_51.txt [hunk 3]
```diff
@@ -147,7 +147,7 @@ will be used, which is a backslash.  Then a map for "\a" will be defined.
 Note that <unique> is used, this will cause an error message if the mapping
 already happened to exist. |:map-<unique>|
 
-But what if the user wants to define his own key sequence?  We can allow that
+But what if the user wants to define their own key sequence?  We can allow that
 with this mechanism: >
```
Comment (k-takata) [inline runtime/doc/usr_51.txt]:
```suggestion
But what if the user wants to define their own key sequence?  We can allow
that with this mechanism: >
```

**Comment by errael** [comment]
To those interested, one way or the other, in the topic of gender neutral language there is #13509.

**Pull request merged**